app/src/main/res/layout/content_fragment_home.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/accent">
+
+    <RelativeLayout
```

Comment:
Let's prefer LinearLayout over RelativeLayout. Or is it done for some specific reason, can you share the reason with some references?

Reply:
Okay I'll change it to `Linearlayout`. I used`RelativeLayout` because I'm more comfortable with it.

@miPlodder out of curiosity, is there any reason to use `LinearLayout` instead of `RelativeLayout`?

Reply:
Done.